README.md

```diff
@@ -239,6 +239,30 @@ The device plugin will initially discover all PCI network resources in the host
 4. "pfNames" - The Physical funtion name
 5. "linkTypes" - The link type of the net device associated with the PCI device.
 
+### Using UDEV Rules for changing NIC's name:
+
+UDEV rules allow the user to change the NIC's name every time the node get rebooted:
```

Comment:
lets add a preface to this section:

To allow "pfNames" selector to consistently select a specific device across nodes it is required to define a fixed network device name.

UDEV rules allow the user to change the NIC's network device name ....